### PR TITLE
Bump to Release 1.1.0

### LINKED ISSUE #148: Release Astronomer Providers 1.1.0

### PR DESCRIPTION
**Pull request by phanikumv**
This PR includes changes for the Astronomer Providers Release 1.1.0 and includes the below items:

- Databricks triggerer fails to check databricks job state
- Improved exception handling on the Async Databricks Operator
- Documentation changes for the release

Closes #148

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+1.1.0 (2022-03-23)
```
Comment (kaxil) [inline CHANGELOG.rst:4]: We can merged `1.1.0a2` section completely now with only 1.1.0

Comment (kaxil) [reply CHANGELOG.rst:4]: Should we also include `BigQueryTableExistenceSensorAsync` in the list of new operators that was merged today - https://github.com/astronomer/astronomer-providers/pull/135

Comment (phanikumv) [reply CHANGELOG.rst:4]: The example DAG for this sensor is currently failing integration test (it ran fine in local though), hence didn't put it yet.

Comment (phanikumv) [reply CHANGELOG.rst:4]: Included BigQueryTableExistenceSensorAsync and fix to malformed example DAG URLs in the latest commit

**Review by kaxil — APPROVED** [review]

**Pull request merged**